recipe/meta.yaml

```diff
@@ -25,7 +23,7 @@ requirements:
     - configparser  # [py27]
     - future >=0.14  # [py27]
     - backports.functools_lru_cache  # [py27]
-    - jedi >=0.14.1,<0.16
+    - jedi >=0.14.1,<0.16.0a0
```

Comment:
Please remove this. The PyLS doesn't work with any Jedi 0.16 version.

Reply:
right: that's what that's saying: not even pre-releases of 0.16. just to be sure.

Reply:
Ahh, ok, sorry.